src/crypto/blake2/blake2-endian.h

```diff
@@ -44,6 +44,8 @@
 #if !defined( __LITTLE_ENDIAN) && defined(__DARWIN_LITTLE_ENDIAN)
 #define __LITTLE_ENDIAN __DARWIN_LITTLE_ENDIAN
 #endif
+#elif defined(LEDGER_VAULTHSM)
+#include <machine/endian.h>
```

Comment:
Is that OK in native mode?

Reply:
Cannot test it but it compiles So I guess yes :D we will test in native once the PR is merged